src/compiler/csharp_generator_helpers.h

```diff
@@ -25,9 +25,10 @@
 namespace grpc_csharp_generator {
 
 inline bool ServicesFilename(const grpc::protobuf::FileDescriptor* file,
+                             const std::string file_suffix,
```

Comment:
style nit: make this a const reference param?

Reply:
Done